### ISSUE TITLE
Running multiple Masternodes in a single Wallet

### ISSUE THREAD

**Issue by D3m0nKingx** [CoinProjects/AmsterdamCoin#4]
Hello I'm trying to get mutiple masternodes running within the same wallet, I did some googling and found normally the conf file shouldn't have any masternode parameters when running more than one masternode, but rather the masternode.conf file takes over. 

Could you please assist as to what is the correct way to set this up with your coin

My current setup is as follows:

```bash
listen=1
server=1
daemon=1
logtimestamps=1
txindex=1
detachdb=1
enableaccounts=1
staking=0
mnconflock=0
maxconnections=500
rpcuser=demon
rpcpassword=WEYawm8lybTrUAPdhkC9d5e0Eonasa0NoW
externalip=<ip address>
port=10559
rpcport=10560
rpcconnect=127.0.0.1
rpcallowip=127.0.0.1
```
and the masternode.conf:
```bash
mn01 <ip address>:10559 2s4YXvUVvdhLPQu4ttNPPcugtnrkPhmEoJokzTUffrD3HeLsMsN 680921904af4d5174e18b832f378d10a82d258e39d122e63799ffac5cb7ecd8d 0
mn02 <ip address>:10559 2rPHQj843Nebd4PiDUa5bnvaB9iStrDh62dbpJ4hWbo1f81J362 be019733909a63d85b156cc261c3caa65eeba8dd34d0f8289d60abb2ba80f5d6 0
mn03 <ip address>:10559 2rEhBJgW8vQbDaceCEb1AghqLcTCy9sjwtg3i9SgvfmVTuWSipr e9f81b12ab72e17f3c646278a6adde50d0908c04d54017ffb2d3de916177314b 1
mn04 <ip address>:10559 2sjznCRbMQKmYxjgpxNJRAUsJZGHLaJvoYj4vZMzyaWg4j8oS2s ac687d0696d25c4886243b9fde23bcad7104bc01954a85a0631613dd32d942f7 1
```

This setup, the wallet says the masternodes are started, however on the masternode  <network> tab, the  <active (secs)> values always stay at 00m:00s

However setting up the masternode with the conf parameters: 
```bash
masternode=1
masternodeprivkey=2rEPkeh... 
masternodeaddr=<ip address>:port
```
 then the active seconds on the node tab begin to count the time from the start of the node, but this method doesn't work for more than one masternode.

How do I make this work for two or more MNs?

**Comment by PayBitcoin** [comment]
Did you solved the problem?

**Comment by D3m0nKingx** [comment]
Not really, as I've only been able to run a wallet with only one masternode (which is fine in itself), however I thought it would be possible to run more than one masternode from a single wallet, but this requires having more than one privkey defined in the config file. and from what I see this it doesn't work this way, and it has to be separately running nodes (separate confs, separate wallet.dat)
